src/logger/index.js

```diff
@@ -34,37 +35,47 @@ const logger = winston.createLogger({
 	],
 });
 
-
-const colors = {
-//	emerg: 'green',
-//	alert: 'green',
-//	crit: 'green',
-//	error: 'green',
-//	warning: 'green',
-	notice: 'green',
-	info: 'green',
-//	debug: 'green',
-};
+// add events log for methods
+logger.event = event(logger, ['create', 'remove', 'update', 'patch']);
 
 const systemLogger = winston.createLogger({
-	level: 'info',
+	level: systemLogLevel,
+	levels: {			// todo syslog levels ?
+		error: 0,
+		warn: 1,
+		info: 2,
+		http: 3,
```

Comment:
Finde http ein komisches loglevel, finde dazu auch nichts in der winston doku

Reply:
Ok hab es gefunden, aber bei allem anderen seiten wenn ich nach syslog loglevel suche liefert ein anderes Ergebnis. Kann mir also nicht vorstellen, dass das hier Standard konform ist